akka-http/src/main/scala/akka/http/javadsl/server/directives/HeaderDirectives.scala

```diff
@@ -34,7 +35,11 @@ abstract class HeaderDirectives extends FutureDirectives {
    * @group header
    */
   def checkSameOrigin(allowed: HttpOriginRange, inner: jf.Supplier[Route]): Route = RouteAdapter {
```

Comment:
But in the javadsl there is no `HttpOriginRange.Default`. That is why there is a cast below from java's `HttpOriginRange` to scala's `HttpOriginRange.Default`